src/v2/view-builder/views/TerminalView.js

```diff
@@ -20,7 +20,6 @@ const EMAIL_AUTHENTICATOR_TERMINAL_KEYS = [
 
 const GET_BACK_TO_SIGN_LINK_FLOWS = [
   RETURN_LINK_EXPIRED_KEY,
-  UNLOCK_ACCOUNT_TERMINAL_KEY
 ];
```

Comment:
I did not want to change this back to a variable (from an array) as this scales well for multiple screens in future.